sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/http/ReactorNettyClient.java

```diff
@@ -50,8 +55,15 @@ private ReactorNettyClient() {}
     public static ReactorNettyClient create(HttpClientConfig httpClientConfig) {
         ReactorNettyClient reactorNettyClient = new ReactorNettyClient();
         reactorNettyClient.httpClientConfig = httpClientConfig;
-        reactorNettyClient.httpClient = reactor.netty.http.client.HttpClient.newConnection();
+        reactorNettyClient.httpClient = reactor.netty.http.client.HttpClient
+            .newConnection()
+            .observe(getConnectionObserver())
+            .resolver(DefaultAddressResolverGroup.INSTANCE);
         reactorNettyClient.configureChannelPipelineHandlers();
+        //  TODO: (kuthapar) - Un-comment the below line once we upgrade to reactor-netty version 1.0.3
+        //  Related PR - https://github.com/reactor/reactor-netty/pull/1455
+        //  This enables fast warm up of HttpClient
+        //  reactorNettyClient.httpClient.warmup().block();
```

Comment:
Do we have a corresponding GitHub issue filed in the Java repo?

Reply:
Good question, this was copied from another branch where @kushagraThapar commit this change

Reply:
@alzimmermsft - we can uncomment that line, and get rid of the rest of the comments. 
```suggestion
        //  This enables fast warm up of HttpClient
        //  reactorNettyClient.httpClient.warmup().block();
```